openshift_cli_installer/libs/clusters/aro_cluster.py

```diff
@@ -0,0 +1,248 @@
+import os
+import yaml
+import base64
+from clouds.azure.utils import get_aro_supported_versions
+from clouds.azure.session_clients import (
+    azure_credentials,
+    aro_client,
+    network_client,
+    resource_client,
+)
+from openshift_cli_installer.utils.general import random_resource_postfix
+from openshift_cli_installer.libs.clusters.ocp_cluster import OCPCluster
+from simple_logger.logger import get_logger
+
+
+class AROCluster(OCPCluster):
+    def __init__(self, ocp_cluster, user_input):
+        super().__init__(ocp_cluster=ocp_cluster, user_input=user_input)
+        self.logger = get_logger(f"{self.__class__.__module__}-{self.__class__.__name__}")
+        self.set_azure_authentication()
+
+        if not self.user_input.create:
```

Comment:
self.create

and reverse the logic - positive condition

Reply:
only `user_input` has the create attribute.
I can add `self.create = self.user_input.create` on `ocp_cluster.py` init if needed, let me know